test/fixtures/poro.rb

```diff
@@ -4,11 +4,11 @@ def initialize(hash={})
   end
 
   def cache_key
-    "#{self.class.name.downcase}/#{self.id}-#{self.updated_at}"
+    "#{self.class.name.downcase}/#{self.id}-#{self.updated_at.strftime("%Y%m%d%H%M%S%9N")}"
   end
 
   def updated_at
-    @attributes[:updated_at] ||= DateTime.now.to_time.to_i
```

Comment:
This usage of `#to_i` was one of the root causes behind the tests passing when they shouldn't have, and (incidentally) the tests not testing for real-world scenarios where `updated_at` isn't an integer, but is an instance of `Time` (or more likely an `ActiveSupport::TimeWithZone` instance.)